docs/Lesson1.md

```diff
@@ -138,7 +138,7 @@ OK, let's go:
         ? "Model: ", device_info.getModel()
         ? "Display Name: ", device_info.getModelDisplayName()
         ? "Firmware: ", device_info.getVersion()
-        ? "Device ID: ", device_info.getDeviceUniqueId()
+        ? "Device ID: ", device_info.GetChannelClientId()
```

Comment:
Is `getDeviceUniqueId` no longer supported? Just curious.

Reply:
ah, found this: ```GetDeviceUniqueID() is deprecated and will be removed from the firmware in the Spring 2019 OS update. Developers must update their channels to use the 12-character alphanumeric unique identifier returned by GetChannelClientId()```

https://community.roku.com/t5/Roku-Developer-Program/serial-number-of-the-device/m-p/504221